lib/godwoken_indexer/worker/check_update_transaction_method_id_name.ex

```diff
@@ -82,6 +83,7 @@ defmodule GodwokenIndexer.Worker.CheckUpdateTransactionMethodIdName do
       where: a.type == :polyjuice_contract,
       inner_join: p in Polyjuice,
       on: t.hash == p.tx_hash,
+      where: fragment("length(?)", p.input) > 0,
```

Comment:
直接input_size != 0 即可
```suggestion
      where: p.input_size != 0,
```